package_manager/dpkg_parser.py

```diff
@@ -0,0 +1,99 @@
+import argparse
```

Comment:
I think we need a copyright string up here, also a file-level comment explaining this.